### DISCUSSION TITLE
How do I add multiple `.pdf` files for a single entry/citekey?

### DISCUSSION THREAD

**Discussion by deen1** [emacs-citar/citar#820] [Q&A]
I've set the `citar-file-additional-files-separator` variable, but when I try to do `M-x citar-add-file-to-library` for an entry that already has a file, I get given the "File ... exists, overwrite? (y or n)" prompt in the minibuffer.

Or is the case that new files have to be manually renamed and moved into the same directory as the existing files?

**Comment by bdarcus** [comment]
Am not sure, since I use neither of these features, but it's possible that command isn't designed with that feature in mind ATM?

If so, we should address that.

EDIT: looking at the source code for the default function for that, my hunch is correct.

https://github.com/emacs-citar/citar/blob/885b86f6733fd70f42c32dd7791d3447f93db990/citar.el#L1337-L1360

Feel free to add an enhancement request to the issue tracker, perhaps with some detail of how that logic should work?

**Reply by deen1** [reply]
OK, have done so. I realize that I'm still on an older version of `citar` (1.3.1)  as I couldn't find the function you've described in `citar.el`.
